rave-test/src/main/java/com/uber/rave/AnnotationSpecs.java

```diff
@@ -192,6 +203,18 @@ public Builder setIntDef(int... values) {
             return this;
         }
 
+        /**
+         * Set the valid values for the long in a {@link LongDef}.
+         *
+         * @param values the valid values.
+         * @return this {@link Builder}
+         */
+        public Builder setLongDef(long ... values) {
+            annotationSpecs.hasLongDef= true;
```

Comment:
Please fix the code style in this method

Reply:
Good catch. dfa33e7